compass/ocean/tests/spherical_harmonic_transform/qu_convergence/mesh.py

```diff
@@ -0,0 +1,140 @@
+from mpas_tools.logging import check_call
+
+from compass.step import Step
+from compass.model import make_graph_file
+
+import os
+
+
+class Mesh(Step):
+    """
+    A step for creating uniform global meshes
+
+    Attributes
+    ----------
+    resolution : int
+        The resolution of the (uniform) mesh in km
+
+    serial_nLat : int
+        The number of latitudes in the Gaussian grid
+    """
+    def __init__(self, test_case, resolution, serial_nLat):
+        """
+        Create a new step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.spherical_harmonic_transform.qu_convergence
+            The test case this step belongs to
+
+        resolution : int
+            The resolution of the (uniform) mesh in km
+
+        serial_nLat : int
+            The number of latitudes in the Gaussian grid
+
+        """
+        super().__init__(test_case=test_case,
+                         name=f'QU{resolution}_mesh',
+                         subdir=f'QU{resolution}/mesh',
+                         cores=18, min_cores=1, threads=1)
+        for file in ['mesh.nc', 'graph.info']:
+            self.add_output_file(filename=file)
+
+        self.resolution = resolution
+        self.serial_nLat = serial_nLat
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        logger = self.logger
+
+        # only use progress bars if we're not writing to a log file
+        use_progress_bar = self.log_filename is None
+
+        # create the base mesh
+        cellWidth, lon, lat = self.build_cell_width_lat_lon()
+        build_spherical_mesh(cellWidth, lon, lat, out_filename='mesh.nc',
+                             logger=logger, use_progress_bar=use_progress_bar)
+
+        make_graph_file(mesh_filename='mesh.nc',
+                        graph_filename='graph.info')
+
+        for nLat in self.serial_nLat:
+            self.mapping_files(nLat)
+
+    def build_cell_width_lat_lon(self):
+        """
+        Create cell width array for this mesh on a regular latitude-longitude
+        grid
+        Returns
+        -------
+        cellWidth : numpy.array
+            m x n array of cell width in km
+
+        lon : numpy.array
+            longitude in degrees (length n and between -180 and 180)
+
+        lat : numpy.array
+            longitude in degrees (length m and between -90 and 90)
+        """
+        dlon = 10.
+        dlat = dlon
+        constantCellWidth = float(self.resolution)
+
+        nlat = int(180/dlat) + 1
+        nlon = int(360/dlon) + 1
+        lat = np.linspace(-90., 90., nlat)
+        lon = np.linspace(-180., 180., nlon)
+
+        cellWidth = constantCellWidth * np.ones((lat.size, lon.size))
+        return cellWidth, lon, lat
+
+    def mapping_files(self, N):
+
+        nLat = str(N)
+        nLon = 2*int(N)
+        nLon = str(nLon)
+
+        parallel_executable = self.config.get('parallel', 'parallel_executable')
```

Comment:
This one too seems it like should stay on one line.

Reply:
It really isn't easy to read in the terminal.  I would very much prefer:

```suggestion
        parallel_executable = \
            self.config.get('parallel', 'parallel_executable')
```

or

```suggestion
        parallel_executable = self.config.get('parallel', 
                                              'parallel_executable')
```